package.json

```diff
@@ -13,10 +13,11 @@
     "testcafe"
   ],
   "dependencies": {
-    "saucelabs": "4.6.3",
-    "testcafe": "1.8.5",
-    "webdriverio": "6.1.17",
-    "xml2js": "0.4.23",
-    "js-yaml": "3.14.0"
+    "js-yaml": "^3.14.0",
+    "saucectl": "^0.23.2",
+    "saucelabs": "^4.6.3",
+    "testcafe": "^1.8.5",
```

Comment:
I think the testcafe version should be fixed

Reply:
imho we should fixate all version, since we don't really benefit from the `^` prefix in our usecase (it suggests something that won't and shouldn't happen).

ℹ️ There is never a case where we'll update the minor or even patch version without explicitly doing so and releasing a new bundle.